spec/errors.md

```diff
@@ -46,7 +46,7 @@ or contains some error which leads to further errors,
 an implementation which does not emit all of the errors
 SHOULD prioritise _Syntax Errors_ and _Data Model Errors_ over others.
 
-When an error occurs within a _selector_,
+When an error occurs while resolving a _selector_,
```

Comment:
Perhaps not "while resolving"? My thinking here is that selector resolution means determining what the selector consists of. The original text covered internal errors of the function itself. Perhaps:

```suggestion
When an error occurs during resolution of a _selector_ or within the _selector_,
```

Reply:
No, because parsing and resolution are not what the original problem was. The selector is a function call. If the selector blows up (signals an error, for example by throwing an exception), that has nothing to do with parsing or resolution. This text is meant to deal with that situation. Parsing errors are syntax errors (or data model errors) and happen long before calling the selector. Resolution errors are a potential problem, though.

Reply:
Got it. Thanks. We're talking about [MatchSelectorKeys](https://github.com/unicode-org/message-format-wg/blob/ae120c0b22d8888bfb4df5b232bbe605f73d76f0/spec/formatting.md#resolve-preferences) here (or something very nearby it)?

Reply:
This needs to cover errors in both [Resolve Selectors](https://github.com/unicode-org/message-format-wg/blob/main/spec/formatting.md#resolve-selectors) (step 2.iii.) and in [Resolve Preferences](https://github.com/unicode-org/message-format-wg/blob/main/spec/formatting.md#resolve-preferences) (step 2.iv., calling calling MatchSelectorKeys). Hence also the two sorts of errors that are mentioned below, though that needs an update due to #828 -- I'll include that here.

I don't think "within" works anymore, as _selector_ is now a _variable_. Maybe something like this?

```suggestion
When an error occurs while resolving a _selector_
or calling MatchSelectorKeys with its resolved value,
```